apps/files/models.py

```diff
@@ -597,7 +597,7 @@ def save(self, *args, **kw):
         super(FileUpload, self).save()
 
     def add_file(self, chunks, filename, size):
-        filename = smart_str(filename)
+        filename = smart_str('{0}_{1}'.format(self.pk, filename))
```

Comment:
As far as I can tell we shouldn't seeing as this doesn't seem to affect how we currently search for files. The only use of using the file name as a lookup was in the unit tests (which is not fixed). @magopian may have a better idea however.